webapp/tests/karma/ts/effects/reports.effects.spec.ts

```diff
@@ -93,107 +94,251 @@ describe('Reports effects', () => {
       actions$ = of([
         ReportActionList.markReportRead(''),
         ReportActionList.removeSelectedReport({}),
-        ReportActionList.setSelected({}),
+        ReportActionList.openReportContent({}),
       ]);
       effects.selectReport.subscribe();
+
       expect(reportViewModelGeneratorService.get.callCount).to.equal(0);
     }));
 
     it('should skip when no provided id', waitForAsync(() => {
-      actions$ = of(ReportActionList.selectReport({  }));
+      actions$ = of(ReportActionList.selectReport({}));
+      effects.selectReportToOpen.subscribe();
+
+      expect(reportViewModelGeneratorService.get.notCalled).to.be.true;
+    }));
+
+    it('should skip when no in select mode', waitForAsync(() => {
+      store.overrideSelector(Selectors.getSelectMode, false);
+      store.refreshState();
+
+      actions$ = of(ReportActionList.selectReport({}));
```

Comment:
Please make this action call valid. In the following tests, you're calling the action with a string parameter.

Reply:
Good catch, it should be a string, I've fixed all occurrences with the same mistake.